slackview.php

```diff
@@ -88,7 +88,7 @@ function get_channel_history($channelId,$history_count)
 
     while ($has_more && count($channel_history) < $history_count) {
         $h = slack_api_request('conversations.history', [
-            'channel' => $channelId,
+            'channel' => $slackChannelName,
```

Comment:
The API for the channel field is an Id: https://api.slack.com/methods/conversations.history. A channel name (such as "general") is not valid.

Reply:
It seems showing error after for few days by using $slackChannelName. So i revert back to $channelId and it fine.